log.go

```diff
@@ -26,7 +26,8 @@ const (
 	LogBarrier
 
 	// LogConfiguration establishes a membership change configuration. It is
-	// created when a server is added, removed, promoted, etc.
+	// created when a server is added, removed, promoted, etc. Only used
+	// when protocol version 1 or greater is in use.
```

Comment:
GitHub disallows me from placing this comment where it should go on lines 14 and 18. Those should state "Only used when protocol version whatever is in use".